test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/CSharpEndToEndTests.cs

```diff
@@ -72,10 +72,10 @@ public void MobileTablesTable()
             //await WaitForMobileTableRecordAsync("Item", id);
         }
 
-        [Fact(Skip = "Not yet enabled.")]
-        public void FileLogging_Succeeds()
+        [Fact]
+        public async Task FunctionLogging_Succeeds()
```

Comment:
Renaming because the test is just verifying ILogger calls work. And in our test harness, we're using a TestLogger, so not actually logging to files.